sympy/calculus/tests/test_util.py

```diff
@@ -122,6 +123,15 @@ def test_periodicity():
     assert periodicity(tan(x), y) == S.Zero
 
     assert periodicity(exp(x), x) is None
+    assert periodicity(exp(I*x), x) is None
+    assert periodicity(exp(I*z), z) == 2*pi
+    assert periodicity(exp(z), z) is None
+    assert periodicity(exp(log(sin(z) + I*cos(2*z)), evaluate=True), z) == 2*pi
+    assert periodicity(exp(log(sin(2*z) + I*cos(z)), evaluate=True), z) == 2*pi
```

Comment:
I think these were meant to be `evaluate=False`. `evaluate=True` is already the default.